pkg/ensurance/executor/throttle.go

```diff
@@ -281,7 +281,7 @@ func (t *ThrottleExecutor) Restore(ctx *ExecuteContext) error {
 						bSucceed = false
 						continue
 					}
-					klog.V(2).Infof("restore kkkkkkkk")
+					klog.V(2).Infof("restore")
```

Comment:
log should be a full statement.
and v(2) can be removed per code standard.

Reply:
Ok